### PR TITLE
[docs] Guide migration - index, menu, redirects [WIP]

### PR DESCRIPTION
**Pull request by judithpatudith**
This PR will direct users from the old guides housed on the consul.io website to the newly migrated guides on the Learn platform. To be merged after [this PR](https://github.com/hashicorp/learn/pull/418) migrating the rest of the guides to learn.

### CODE REVIEW

File: website/source/docs/guides/index.html.md [hunk 1]
```diff
@@ -8,60 +8,18 @@ description: |-
...
-* [Outage Recovery](https://learn.hashicorp.com/consul/day-2-operations/advanced-operations/outage) - This guide covers recovering a cluster that has become unavailable due to server failures.
-
-* [Consul Reference Architecture](https://learn.hashicorp.com/consul/advanced/day-1-operations/reference-architecture) - This document provides recommended practices and a reference architecture, including system requirements, datacenter design, networking, and performance optimizations for Consul production deployments.
-
-* [Semaphore](/docs/guides/semaphore.html) - This guide covers using the KV store to implement a semaphore.
-
-* [Server Performance](/docs/guides/performance.html) - This guide covers minimum requirements for Consul servers as well as guidelines for running Consul servers in production.
-
-* [Windows Service](/docs/guides/windows-guide.html) - This guide covers how to run Consul as a service on Windows.
+- [Getting Started]()
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/index.html.md]:
```suggestion
- [Getting Started](https://learn.hashicorp.com/consul/?track=getting-started#getting-started)
```

File: website/source/docs/guides/index.html.md [hunk 2]
```diff
@@ -8,60 +8,18 @@ description: |-
...
+- [Getting Started with Kubernetes]()
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/index.html.md]:
```suggestion
- [Getting Started with Kubernetes](https://learn.hashicorp.com/consul/?track=getting-started-k8s#getting-started-k8s)
```

File: website/source/docs/guides/index.html.md [hunk 3]
```diff
@@ -8,60 +8,18 @@ description: |-
...
+- [Day 1: Deploying Your First Datacenter]()
+- [Day 1: Security and Network Operations]()
+- [Day 1: Kubernetes Operations]()
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/index.html.md]:
```suggestion
- [Day 1: Kubernetes Operations](https://learn.hashicorp.com/consul/?track=kubernetes#kubernetes)
```

File: website/source/docs/guides/index.html.md [hunk 4]
```diff
@@ -8,60 +8,18 @@ description: |-
...
+- [Day 2: Advanced Operations]()
+- [Development]()
+- [Integrations]()
+- [Kubernetes in Production]()
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/index.html.md]: I think Kubernetes in Production is Kubernetes Operations.

File: website/source/docs/guides/index.html.md [hunk 5]
```diff
@@ -8,60 +8,18 @@ description: |-
...
-
-* [Server Performance](/docs/guides/performance.html) - This guide covers minimum requirements for Consul servers as well as guidelines for running Consul servers in production.
-
-* [Windows Service](/docs/guides/windows-guide.html) - This guide covers how to run Consul as a service on Windows.
+- [Getting Started]()
+- [Getting Started with Kubernetes]()
+- [Day 1: Deploying Your First Datacenter]()
+- [Day 1: Security and Network Operations]()
+- [Day 1: Kubernetes Operations]()
+- [Day 2: Advanced Operations]()
```
Comment (kaitlincarter-hc) [inline website/source/docs/guides/index.html.md]:
```suggestion
- [Day 2: Advanced Operations](https://learn.hashicorp.com/consul/?track=day-2-operations#day-2-operations)
```

**Comment by alvin-huang** [comment]
Content looks good to me! You'll need to rebase and resolve those conflicts before merging though 😃

**Comment by judithpatudith** [comment]
Closed in favor of #5761

**Pull request closed without merging**